portfolio/src/main/webapp/information.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Thomas Pena</title>
+    <link rel="stylesheet" href="StyleMenu.css">
+    <script src="script.js"></script>
+        
+  </head>
+
+
+<body>
+<div class="projects">
+ <div id="socials">
+    <p><p1 style="color:black">Thomas Pena</p1> 
+    <button onclick="github()"><i class="fa fa-github"></i></button>
+    <button onclick="linkedin()"><i class="fa fa-linkedin"></i></button>
+    </p>
+ </div> 
+    <h1 style="text-align:center;font-size:30px;">Projects</h1> 
+    <p>  <a onclick="project1()" id="projects"><img src="/images/blackjack.jpg" class="picture1"/></a>
+        <a onclick="project2()" id="projects"><img src="/images/house.jpg" class="picture2"/></a>
+        <a onclick="project3()" id="projects"><img src="/images/blinds.jpg" class="picture3"/></a>        
+    </p>
+    
+    <div id="project1" style="display:none">       
+    <p>
+    Blackjack – Java: Simplified version of the game Blackjack<br>
+    •	Implemented random card drawing, calculation, state tracking, and console input systems<br>
+    •	Data types, console I/O, control structures, and libraries<br>
+    </div> 
+    </p>
+  <div id="project2" style="display:none">       
+   <p>
+        House Renovation – Python: Determined the number of boxes needed for my parent’s house renovation<br>
+        •	Calculated intelligently via final cost several other factors <br>
+            o	Area of the room, number of tiles per box, dimensions of each tile, and different brands
+    </p>
+    </div> 
+    <div id="project3" style="display:none">       
+    <p>
+    Automated Blinds – C++ | Arduino: Voice-controlled automated blinds
+    •	IoT project: Used Adafruit to create an open/close switch, with IFTTT platform 
+        implemented the trigger that changes the switch based on voice commands through Google Assistant 
+
+    </p>
+    </div> 
+</div> 
+
+<div class="experience">
```

Comment:
I would call this `experiences-container` since this is really the container of your actual experience.